### DISCUSSION TITLE
Why this xformers  error?

### DISCUSSION THREAD

**Discussion by leonnn1** [bmaltais/kohya_ss#622] [Q&A]
ERROR: xformers-0.0.14.dev0-cp310-cp310-win_amd64.whl is not a supported wheel on this platform.

I am getting this error and I have no idea why

**Comment by leonnn1** [comment]
I guess this thing just can't work

**Comment by jstayco** [comment]
Sorry, I'm just seeing this, so I know it's not much help, but that error indicates you're trying to install a 64-bit WIndows version of xformers on an operating system that is not 64-bit Windows.
